test/e2e-go/features/catchup/basicCatchup_test.go

```diff
@@ -76,19 +77,22 @@ func TestBasicCatchup(t *testing.T) {
 // The current versions are the original v1 and the upgraded to v2.1
 func TestCatchupOverGossip(t *testing.T) {
 	t.Parallel()
+
+	versions := network.SupportedProtocolVersions
+	require.LessOrEqual(t, len(versions), 3)
+
 	// ledger node upgraded version, fetcher node upgraded version
-	runCatchupOverGossip(t, false, false)
-	// ledger node older version, fetcher node upgraded version
-	runCatchupOverGossip(t, true, false)
-	// ledger node upgraded older version, fetcher node older version
-	runCatchupOverGossip(t, false, true)
-	// ledger node older version, fetcher node older version
-	runCatchupOverGossip(t, true, true)
+	for i := 0; i < len(versions); i++ {
+		runCatchupOverGossip(t, "", "")
```

Comment:
This line doesnt need to be within the loop

Reply:
Will put this outside the loop, and replace it with:
`runCatchupOverGossip(t, supportedVersions[0], supportedV[0])`
The default value from the config file will be empty. So, the value will be picked up from supportedVersions index 0.